CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ project(
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_SUPPRESS_DEVELOPER_WARNINGS 1 CACHE INTERNAL "No dev warnings")
```

Comment:
I was able to fix the cmake warning here: https://github.com/intel-ai/hdk/pull/531

Reply:
Cool) I also tried in the same way, but didn't assume that the extra space would resolve it.